tutorials/early_stopping/early_stopping.ipynb

```diff
@@ -0,0 +1,1067 @@
+      "[INFO 09-08 17:44:58] ax.modelbridge.dispatch_utils: Using Bayesian Optimization generation strategy: GenerationStrategy(name='Sobol+GPEI', steps=[Sobol for 5 trials, GPEI for subsequent trials]). Iterations after 5 will take longer to generate due to  model-fitting.\n"
+     ]
+    }
+   ],
+   "source": [
+    "total_trials = 20  # total evaluation budget\n",
+    "\n",
+    "gs = choose_generation_strategy(\n",
+    "    search_space=experiment.search_space,\n",
+    "    optimization_config=experiment.optimization_config,\n",
+    "    num_trials=total_trials,\n",
+    "  )"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "78d86fea",
+   "metadata": {},
+   "source": [
+    "## Configuring the Scheduler\n",
+    "\n",
+    "The `Scheduler` acts as the loop control for the optimization.\n",
+    "It communicates with the backend to launch trials, check their status, retrieve (partial) results, and importantly for this tutorial, calls the early stopping strategy. If the early stopping strategy suggests a trial to be the stopped, the `Scheduler` communicates with the backend to terminate the trial.\n",
+    "\n",
+    "The ``Scheduler`` requires the ``Experiment`` and the ``GenerationStrategy``.\n",
+    "A set of options can be passed in via ``SchedulerOptions``. Here, we\n",
+    "configure the number of total evaluations as well as ``max_pending_trials``,\n",
+    "the maximum number of trials that should run concurrently. In our\n",
+    "local setting, this is the number of training jobs running as individual\n",
+    "processes, while in a remote execution setting, this would be the number\n",
+    "of machines you want to use in parallel.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "id": "499fb9b5",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "[INFO 09-08 17:44:58] Scheduler: `Scheduler` requires experiment to have immutable search space and optimization config. Setting property immutable_search_space_and_opt_config to `True` on experiment.\n"
+     ]
+    }
+   ],
+   "source": [
+    "scheduler = Scheduler(\n",
+    "    experiment=experiment,\n",
+    "    generation_strategy=gs,\n",
+    "    options=SchedulerOptions(\n",
+    "        total_trials=total_trials, max_pending_trials=3, early_stopping_strategy=percentile_early_stopping_strategy\n",
+    "    ),\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "78257ebb",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "scheduler.run_all_trials()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d6ec43d9",
+   "metadata": {},
+   "source": [
+    "## Results\n",
+    "\n",
+    "Below is a summary of the experiment, showing that a portion of trials have been early stopped."
```

Comment:
the df right after this doesn't actually show that trials were early stopped (that is shown by the trial status in the next one). Maybe move the first one one cell back and add a comment on how that df shows the raw data of the curve observations